tqecjs/src/workspace/index.js

```diff
@@ -94,6 +94,7 @@ export default function TQECApp() {
 		// Create the plaquettes and tile
 		const tile = new Tile(selectedQubits, app);
 		tile.createPlaquette();
+		tile.container.isTile = true;
```

Comment:
What's the difference between the .isTile in the class already?

Reply:
The reason why I added it here is that when I go to download, I traverse the workspace containers, and want to filter on those that are tiles. 
```js
		const tiles = workspace.children.filter((child) => child.isTile);
```